api/src/dto/ContentDto.ts

```diff
@@ -0,0 +1,36 @@
+import { IsArray, IsNotEmpty, IsNumber, IsString } from "class-validator";
+import { ContentStatus, DocType, Uuid } from "../types";
+
+/**
+ * Database structured Content object
+ */
+export class ContentDto {
+    @IsNotEmpty()
+    _id: Uuid;
+    @IsNotEmpty()
+    type: DocType.Content;
+    @IsArray()
+    memberOf: Uuid[];
+    @IsNotEmpty()
+    language: Uuid;
+    @IsNotEmpty()
+    status: ContentStatus;
+    @IsString()
+    slug: string;
+    @IsString()
+    title: string;
+    @IsString()
+    summary: string;
+    @IsString()
+    author: string;
+    @IsString()
+    text: string;
+    @IsString()
+    seo: string;
```

Comment:
Small note, I propose to leave this out for now as I don't think "seo" will actually be a string, more likely a separate class with several fields such as description and keywords. This is what the current site has:

![image](https://github.com/bccsa/ac-app/assets/39056299/e1bda58e-1b4d-46ef-8057-75dd5c4ceda7)

I left it out in the frontend for now for that reason as well.

Reply:
Commented out for now